icicle-compiler/src/Icicle/Sea/FromAvalanche/Base.hs

```diff
@@ -91,7 +91,7 @@ mangleToSeaName = SeaName . Text.pack . concatMap mangle . show . pretty
      | c >= 'A' && c <= 'Z'
      = [c]
      | c == '_'
-     = "__"
+     = "_"
```

Comment:
it may not matter in practice, but the reason I had escaped the underscore here was to distinguish an identifier containing underscores from the underscore followed by hex: `fish_21` vs `fish_21` or `fish!`